src/data/drinks.ts

```diff
@@ -0,0 +1,348 @@
+export interface Drink {
+    category: string;
+    name: string;
+    img?: string;
+    glassIcon?: string;
+    glassPrice?: number;
+    bottleIcon?: string;
+    bottlePrice?: number;
```

Comment:
```suggestion
    glassPrice?: number;
    bottlePrice?: number;
```
No deberías añadir esto a la data. Recuerda, esto es algo que persistes en la base de datos, y no vas a persistir un "path" de lo que hay en un project local. Lo que deberías hacer en el `DrinkCard` si existe un glassPrice, en caso afirmativo añades ese icon